### PR TITLE
Group-IB Threat Intelligence

### PR DESCRIPTION
**Pull request by LanskoyGIB**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by dorschw — DISMISSED** [review]
Please revert all changes outside the `Packs/` folder, they seem unintended.

**Comment by LanskoyGIB** [comment]
Hello,

We reverted the changes outside the Packs folder.

Regards,
Daniil

File: Packs/Whois/Integrations/Whois/test_data/microsocks
Comment (maimorag) [inline Packs/Whois/Integrations/Whois/test_data/microsocks:1]: Revert this file

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json [hunk 1]
```diff
@@ -292,62 +292,110 @@
...
+                    "dateDetected": "2021-01-28T19:37:08+00:00",
+                    "evaluation": {
+                        "admiraltyCode": "A1",
+                        "credibility": 30,
+                        "reliability": 100,
+                        "severity": "gray",
+                        "tlp": "amber",
+                        "ttl": 30
+                    },
+                    "id": "9ea4c62e9e5da6bc86ecb35e9f4da12fd29cfa5e",
```
Comment (maimorag) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json:345]: Or add it to secrets-ignore
```suggestion
                    "id": "1111111111111111111111111111111111111111",
```

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json [hunk 2]
```diff
@@ -292,62 +292,110 @@
...
+                                {
+                                    "bindBy": "",
+                                    "companyId": 0,
+                                    "data": "",
+                                    "ruleId": 0,
+                                    "type": "readme"
+                                }
+                            ],
+                            "data": null,
+                            "hash": "21e02c97e2ecdabb272bfefbe29fd3f50032f525",
```
Comment (maimorag) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json:365]: Or add it to secrets-ignore
```suggestion
                            "hash": "1111111111111111111111111111111111111111",
```

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json [hunk 3]
```diff
@@ -292,62 +292,110 @@
...
+                    "rules": null,
+                    "url": "https://github.com/somegit"
                     }
-                ],
-                "seqUpdate": 1611219371626093,
-                "source": "github"
-            }
+            ],
+            "hideForCompanies": [],
+            "id": "21aed9b86d2e6cbb15180d803a84f6d27f673db4",
```
Comment (maimorag) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json:378]: Or add it to secrets-ignore
```suggestion
            "id": "1111111111111111111111111111111111111111",
```

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA_Feed/test_data/example.json [hunk 4]
```diff
@@ -680,5 +680,34 @@
         "timeLineData": [],
         "title": "CVE-2021-27152",
         "type": "cve"
+    },
+    "ioc/common":{
+        "id": "6d44a7ed0b826f972e2fb74233c581d80f890552",
```
Comment (maimorag) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA_Feed/test_data/example.json:685]: Or add it to secrets-ignore
```suggestion
        "id": "1111111111111111111111111111111111111111",
```

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA_Feed/test_data/example.json [hunk 5]
```diff
@@ -680,5 +680,34 @@
...
+            "dateLastSeen": "2016-10-24T00:00:00Z",
+            "domain": "some.ru",
+            "url": "https://some.ru",
+            "ip": [
+                "11.11.11.11"
+            ],
+           "hash": [
+                "ecbb901449a852d5cd756c17718cf9001c22077f",
+                "f79f5afdb76ce971305e4bfeb5e4c5447156074c2c9c72974af1d3dc83e4b861",
+                "07e266f7fb3c36a1f3a5c5d2d229a478"
```
Comment (maimorag) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA_Feed/test_data/example.json:697]: Or add it to secrets-ignore
```suggestion
                "1111111111111111111111111111111111111111",
                "1111111111111111111111111111111111111111111111111111111111111111",
                "11111111111111111111111111111111"
```

**Review by maimorag — COMMENTED** [review]
Hi @LanskoyGIB,
Thank you for your contribution!
Good work :)

Few notes:
1. Logo should match the requirements (120x50 px PNG, transparent background, <10kB in Size)
2.  Please bump the pack version in the pack_metadata.json file and add notes describing your changes to a new Release Notes file.
To do that, run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.
3. Merge from master and solve the conflict.

**Comment by maimorag** [comment]
@LanskoyGIB please revert the changes of the file Packs/Whois/Integrations/Whois/test_data/microsocks, for example:
`git checkout origin/master -- Packs/Whois/Integrations/Whois/test_data/microsocks`
Also, merge from master and solve the conflicts.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by maimorag** [comment]
Hi @LanskoyGIB, we haven’t heard from you in a while.
Do you need any help with the pull request?

Please feel free to reach out to me here or on Slack.
Thanks again for contributing to our repo, hope to hear from you soon.

**Comment by LanskoyGIB** [comment]
Hello,

Sorry for delay i was on vacation, add new commit with required fix

**Comment by maimorag** [comment]
@LanskoyGIB Please merge from master and solve the conflicts :)

**Comment by maimorag** [comment]
Hey @LanskoyGIB, 
do you need any help with the pull request?

Also, please [sign the CLA](https://github.com/demisto/content/pull/28261#issuecomment-1640218941)

**Comment by LanskoyGIB** [comment]
Hello,

Signed the CLA, still on it. Will send you update commit tomorrow.

Regards,
Daniil

**Comment by LanskoyGIB** [comment]
Should i open new PR from master branch?

**Comment by LanskoyGIB** [comment]
Hello,

We are created new PR from master branch.
https://github.com/demisto/content/pull/29350

**Pull request closed without merging**